src/os/bluestore/BlueStore.cc

```diff
@@ -6454,6 +6478,10 @@ int BlueStore::mkfs()
   if (r < 0)
     goto out_close_fsid;
 
+  if (bdev->is_smr()) {
```

Comment:
May be better do that inside _open_bdev?

Reply:
Done.